updater/versions/update-1.py

```diff
@@ -272,6 +272,13 @@ def _perform_update():
 
     shutil.copyfile("/usr/share/zoneinfo/UTC", "/etc/localtime")
 
+    # Fix pip bug introduced in setuptools v34.0.0
+    # http://setuptools.readthedocs.io/en/latest/history.html#v34-0-0
+    check_call(["apt-get", "-y", "install", "-t", "stretch", "python-pip"])
+    # Remove previously installed python packages
+    check_call(["apt-get", "-y", "autoremove"])
+    check_call(["pip", "install", "--upgrade", "setuptools"])
```

Comment:
We should explicitly specify the version number of setuptools with `setuptools==KNOWN_WORKING_VERSION` and omit `--upgrade`